lib/config/3-x-recommended.js

```diff
@@ -43,6 +42,7 @@ export default {
     'no-outlet-outside-routes': 'error',
     'no-partial': 'error',
     'no-passed-in-event-handlers': 'error',
+    'no-pointer-down-event-binding': 'error',
```

Comment:
I believe the `3-x-recommended.js` config was specific to version 3.0 and is no longer in use/relevant, so this change isn't necessary.